### PR TITLE
website/docs: update 2023.5 release notes

### PR DESCRIPTION
**Pull request by BeryJu**
# Details

Update 2023.5 release notes with postgresql 11 deprecation, and update changelog

### New Features

-   Adds feature which does x, y, and z.

### Breaking Changes

-   Adds breaking change which causes \<issue\>.

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/releases/2023/v2023.5.md [hunk 1]
```diff
@@ -37,8 +41,4616 @@ image:
...
+Changed response : **200 OK**
+
+-   Changed content type : `application/json`
+
+    New required properties:
+
+    -   `managed`
+
+    *   Changed property `managed` (string)
+        > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:100]:
```suggestion
        > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 2]
```diff
@@ -37,8 +41,4616 @@ image:
...
+
+##### `PUT` /crypto/certificatekeypairs/{kp_uuid}/
+
+###### Request:
+
+Changed content type : `application/json`
+
+-   Deleted property `managed` (string)
+    > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:109]:
```suggestion
    > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 3]
```diff
@@ -37,8 +41,4616 @@ image:
...
+Changed response : **200 OK**
+
+-   Changed content type : `application/json`
+
+    New required properties:
+
+    -   `managed`
+
+    *   Changed property `managed` (string)
+        > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:122]:
```suggestion
        > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 4]
```diff
@@ -37,8 +41,4616 @@ image:
...
+
+##### `PATCH` /crypto/certificatekeypairs/{kp_uuid}/
+
+###### Request:
+
+Changed content type : `application/json`
+
+-   Deleted property `managed` (string)
+    > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:131]:
```suggestion
    > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 5]
```diff
@@ -37,8 +41,4616 @@ image:
...
+Changed response : **200 OK**
+
+-   Changed content type : `application/json`
+
+    New required properties:
+
+    -   `managed`
+
+    *   Changed property `managed` (string)
+        > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:144]:
```suggestion
        > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 6]
```diff
@@ -37,8 +41,4616 @@ image:
...
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:159]:
```suggestion
        > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 7]
```diff
@@ -37,8 +41,4616 @@ image:
...
+        > -   `any` - any, any policy must pass
+
+##### `POST` /crypto/certificatekeypairs/
+
+###### Request:
+
+Changed content type : `application/json`
+
+-   Deleted property `managed` (string)
+    > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:527]:
```suggestion
    > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 8]
```diff
@@ -37,8 +41,4616 @@ image:
...
+Changed response : **201 Created**
+
+-   Changed content type : `application/json`
+
+    New required properties:
+
+    -   `managed`
+
+    *   Changed property `managed` (string)
+        > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:540]:
```suggestion
        > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 9]
```diff
@@ -37,8 +41,4616 @@ image:
...
+    -   Changed property `results` (array)
+
+        Changed items (object): > CertificateKeyPair Serializer
+
+        New required properties:
+
+        -   `managed`
+
+        *   Changed property `managed` (string)
+            > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:559]:
```suggestion
            > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

File: website/docs/releases/2023/v2023.5.md [hunk 10]
```diff
@@ -37,8 +41,4616 @@ image:
...
+    > -   `azuread` - Azure AD
+    > -   `discord` - Discord
+    > -   `facebook` - Facebook
+    > -   `github` - GitHub
+    > -   `google` - Google
+    > -   `mailcow` - Mailcow
+    > -   `openidconnect` - OpenID Connect
+    > -   `okta` - Okta
+    > -   `patreon` - Patreon
+    > -   `reddit` - reddit
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.5.md:1000]: Normally Reddit is capitalized.. is there any reason we don't want to here?

**Review by tanberry — APPROVED** [review]
Offff I gave up around line 1450 of auto-generated text, lol. We can choose how picky we want to be about things like capitalizing SCIM, Reddit, etc.

File: website/docs/releases/2023/v2023.5.md (hunk 1 above)
Comment (BeryJu) [reply website/docs/releases/2023/v2023.5.md:100]: The source text for this is here https://github.com/goauthentik/authentik/blob/8ccdbdc370defc14c3907d3a24ce4941109df8c7/authentik/blueprints/models.py#L32, probably better to edit it there and re-generate the api changes

**Pull request merged**